### ISSUE TITLE
Help Displaying R Squared value and MSE Statistic

### ISSUE THREAD

**Issue by osedej** [osedej/Thesis#2]
Gentlemen,

    I have attached my code for the neural network, GBM, and the linear regression models. I used both these ML models for each individual parameters as well as using all of them since it will help drive discussion in my results. Unfortunately, I am having trouble displaying an R^squared and an MSE for each. After a long time scrubbing forums I figured I would ask to see if anyone has any insight on this. I appreciate any help in advance!

**Comment by osedej** [comment]
The code the above refers to is the file Copy_of_DNN_Thesis and GBM_Cleaned_Thesis

**Comment by twsleight** [comment]
just letting you know I saw this. Do you just need goodness of fit statistics?

**Comment by osedej** [comment]
Sir, thank you. Yes, goodness of fit statistics would be perfect!

**Comment by twsleight** [comment]
Are you just trying to display a value that you already have or do you need to compute it?

**Comment by osedej** [comment]
I need to compute it, If you look at the code I was able to display the MAE, however I also wanted to display the MSE and the Rsquared as well since other literature in this space does so. I have tried multiple ways to do this by scrubbing forums, but it seems to always return errors. For some reason the MSE script that I use states that the function does not exist. For the R^squared value I am not sure how to display it as the predicted values and actual values have different amounts of values within them so it returns an error as well when I try to use the r square metric. I've tried with multiple libraries as well.

**Comment by twsleight** [comment]
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html

**Comment by twsleight** [comment]
how about sklearn? let me know if it throws you an error

**Comment by osedej** [comment]
Thank you sir, I will apply it and will let you know!
